dmoj/cptbox/_cptbox.pyx

```diff
@@ -83,7 +83,6 @@ cdef extern from 'ptbox.h' nogil:
 
     cdef int PTBOX_EXIT_NORMAL
     cdef int PTBOX_EXIT_PROTECTION
-    cdef int PTBOX_EXIT_SEGFAULT
```

Comment:
Why are we deleting this right now?

Otherwise code looks fine and explanation makes sense.

Reply:
https://github.com/DMOJ/judge-server/pull/1061/commits/b62a61496e050ff74ee8d90d95905fd4a3a7a084

Reply:
Sure, makes sense. Honestly would've expected this in a separate PR.